watertap/examples/flowsheets/case_studies/activated_sludge/ASM1_flowsheet.py

```diff
@@ -115,7 +115,7 @@ def build_flowsheet():
     m.fs.MX6 = Mixer(property_package=m.fs.props, inlet_list=["clarifier", "reactor"])
     # Product Blocks
     m.fs.Treated = Product(property_package=m.fs.props)
-    m.fs.Sludge = Product(property_package=m.fs.props)
```

Comment:
I suppose you removed this since the sludge will be the AD inlet in our subset flowsheet. We could later consider having the option of adding this Product block for sludge for someone interested in analyzing the AS process in isolation, or skipping this in the case where we extend to the "full" flowsheet.

Reply:
Yes, However this product stream should be added after the dewatering unit model